packages/nuxt-locokit/.gitignore

```diff
@@ -34,7 +34,12 @@ coverage
 .nyc_output
 
 # VSCode
-.vscode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
```

Comment:
I still don't think these files should be committed.

Reply:
Well, that's `nuxt` who create this file.
So, maybe they need to ? Or at least, it could be useful for other devs ? (don't really know)

Reply:
hmmm, I have doubt. I don't want to block this MR with this.